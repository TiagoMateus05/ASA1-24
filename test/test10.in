7 15
4 4 2 1 6 7 1
7 2 1 2 5 6 7
4 5 4 5 3 6 1
2 2 7 3 1 1 4
5 7 1 7 4 6 1
5 6 4 1 4 5 4
4 4 4 4 5 2 4
4 1 1 6 4 7 6 1 2 1 4 7 6 6 1
5