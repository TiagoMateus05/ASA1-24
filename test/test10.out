1
((((((((((((((4 1) 1) 6) 4) 7) 6) 1) 2) 1) 4) 7) 6) 6) 1)
